manifests/profile/dns/standard.pp

```diff
@@ -9,8 +9,29 @@
 # @example
 #   include nebula::profile::dns::standard
 class nebula::profile::dns::standard {
-  class { 'resolv_conf':
-    nameservers => lookup('nebula::resolv_conf::nameservers'),
-    searchpath  => lookup('nebula::resolv_conf::searchpath'),
+  exec { 'restart_networking':
+    command     => '/bin/systemctl restart networking',
+    refreshonly => true,
```

Comment:
Since this is only ever called from inside the `if` condition, would it make sense to put this in there as well? That would also keep it off our own boxes, so that it doesn't somehow get called on any of them.

Reply:
The location was intentional, but you make a good point about polluting the environment. I'll change it.